solutions/silver/cses-1650.mdx

```diff
@@ -103,4 +103,19 @@ public class RangeXorQueries {
 
 </JavaSection>
 
+<PySection>
```

Comment:
please do different prs for different sols

Reply:
I don't understand what you said here. Can you be specific?

Reply:
Ok. I will do my best to come up with unique solutions